minigalaxy/installer.py

```diff
@@ -152,6 +155,32 @@ def move_and_overwrite(game, temp_dir, target_dir):
     shutil.rmtree(temp_dir, ignore_errors=True)
     return error_message
 
+def create_desktop_file(game):
+    error_message = ""
+    if game.platform == "linux":
+        desktop_file_path = os.path.join(DESKTOP_DIR, game.name+".desktop")
+        # Create desktop file
+        desktop_context = {
+            "game_bin_path":os.path.join(game.install_dir,'start.sh'),
```

Comment:
start.sh is not the only way we use to start a game. We should import proper launch command from launcher instead.